recovery/recovery.go

```diff
@@ -0,0 +1,282 @@
+package recovery
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/orbs-network/scribe/log"
+)
+
+//const DDMMYYYYhhmmss = "2006-01-02-15:04:05"
+const (
+	e_zero_content   = "e_zero_content"
+	e_no_bash_prefix = "e_no_bash_prefix"
+	//e_content_not_changed = "e_content_not_changed"
+)
+
+/////////////////////////////////////////////////
+// JSON
+// {
+// 	"shell": {
+// 	"bin": "bash",
+// 	"run": [
+// 		"https://raw.githubusercontent.com/amihaz/staging-deployment/main/boyar_recovery/shared/disk_cleanup_1",
+// 		"https://raw.githubusercontent.com/amihaz/staging-deployment/main/boyar_recovery/shared/docker_cleanup_1"
+// 	]
+//   }
+// }
+type Shell struct {
+	Bin string   `json:"bin"`
+	Run []string `json:"run"`
+}
+
+type Instructions struct {
+	Shell Shell `json:"shell"`
+}
+
+type Config struct {
+	IntervalMinute uint
+	Url            string
+}
+
+type Recovery struct {
```

Comment:
If we change the ticker to tick every few hours, make sure it's very easy to know from the Status when is the next execution planned

Reply:
it is easy as status shows both last and interval - i can change to to nextTick